requirements/base.txt

```diff
@@ -20,12 +20,13 @@ django-cors-headers~=3.0.2  # https://github.com/ottoyiu/django-cors-headers
 # ------------------------------------------------------------------------------
 djangorestframework~=3.9.4  # https://github.com/encode/django-rest-framework
 djangorestframework-jwt~=1.11.0  # https://github.com/GetBlimp/django-rest-framework-jwt
+djangorestframework_simplejwt~=4.3.0  # https://github.com/davesque/django-rest-framework-simplejwt
 coreapi~=2.3.3  # https://github.com/core-api/python-client
 drf-yasg~=1.15.0  # https://github.com/axnsan12/drf-yasg
 
 # REST auth
 # ------------------------------------------------------------------------------
-django-rest-auth~=0.9.5  # https://github.com/Tivix/django-rest-auth
+git+https://github.com/open-craft/django-rest-auth  # fork of https://github.com/Tivix/django-rest-auth with simpleJWT support
```

Comment:
> Could you pin this to a commit id please?

Sure, that makes sense :+1: 

> Why was this fork necessary?

The implementation of JWT used by [django-rest-auth](https://github.com/Tivix/django-rest-auth) is [no longer maintained ](https://github.com/jpadilla/django-rest-framework-jwt/issues/484). There is [a fork](https://github.com/ankurpandeyvns/django-rest-auth) of the django-rest-auth repository that adds SimpleJWT support (which has the token refreshing fixed).
I forked this to our GH organization, because forks tend to go missing sometimes and a couple year ago I had this exact problem with disappearing Python library (`django-pagination`), to which there were no mirrors at the point. So I just did this to keep it safe.